### ISSUE TITLE
精度

### ISSUE THREAD

**Issue by swjtulinxi** [zrl4836/ACFNet#3]
你好，请问你是用几张gpu跑出的效果啊，

**Comment by zrl4836** [comment]
没有空闲显卡完整跑过

**Comment by zrl4836** [comment]
> 你好，请问你是用几张gpu跑出的效果啊，

没有空闲显卡完整跑过

**Comment by swjtulinxi** [comment]
# multiply & normalize ->(B, N, C')
        energy = torch.bmm(proj_query, proj_key)
        energy_new = torch.max(energy, -1, keepdim=True)[0].expand_as(energy) - energy
        attention = self.softmax(energy_new)
你好，就是中间这句代码torch.max这句我不是太明白，请问他的作用是什么？

**Comment by zrl4836** [comment]
> energy_new = torch.max(energy, -1, keepdim=True)[0].expand_as(energy) - energy

Prevent loss divergence during training

**Comment by zrl4836** [comment]
> # multiply & normalize ->(B, N, C')
> ```
>     energy = torch.bmm(proj_query, proj_key)
>     energy_new = torch.max(energy, -1, keepdim=True)[0].expand_as(energy) - energy
>     attention = self.softmax(energy_new)
> ```
> 
> 你好，就是中间这句代码torch.max这句我不是太明白，请问他的作用是什么？

Prevent loss divergence during training
